app/controllers/licences.controller.js

```diff
@@ -23,7 +23,17 @@ async function selectReturnStartDate (request, h) {
   })
 }
 
+async function requirementsApproved (request, h) {
+  const { id } = request.params
+
+  return h.view('return-requirements/requirements-approved.njk', {
+    activeNavBar: 'search',
+    licenceId: id
+  })
+}
+
 module.exports = {
   noReturnsRequired,
-  selectReturnStartDate
+  selectReturnStartDate,
+  requirementsApproved
```

Comment:
To help those who get anxious about this kind of thing and to avoid the rest of us having to expend any effort we try and have some guidance on how to order things.

We have one for the [top of JS files](), and when I find the time I'll write up ones for [functions](https://github.com/DEFRA/water-abstraction-team/issues/92) and [require() ordering](https://github.com/DEFRA/water-abstraction-team/issues/93).

The long and short of it is that we go alphabetically.

```suggestion
  requirementsApproved,
  selectReturnStartDate
```

So, just a minor tweak. 😁

Reply:
no problem, updated